### ISSUE TITLE
[TD] Skirmish enemy AA defenses always powered

### ISSUE THREAD

**Issue by Gerwin2k** [TheAssemblyArmada/Vanilla-Conquer#898]
![no power down](https://user-images.githubusercontent.com/39087372/221088190-2b61a6bc-0fd2-4cbe-9c27-cb31cc1ce4c8.png)

Build 8eb5185, Win32 GCC version.
Playing a skirmish game. In attached image my orca units destroy the only power plant in an enemy NOD base. 
(The enemy had build one power plant and two obelisks, which was not very logical either)
The SAM site is operational before the power plant destruction, and remains operational afterwards as well. 
Similar situation with the GDI advanced guard tower. 
Player's own AA defenses seem to power down when appropriate.

These skirmish enemy defenses used to power down, in older builds of VanillaTD, like from six months ago.

**Comment by giulianobelinassi** [comment]
Check for a rules.ini in the VC folders and delete it. Obelisk spawning should not happens with Nod anymore since 2988d5c7c4d48cd0 . If out of power the AI should schedule to build a Adv. Power Plant with the highest priority possible.

**Comment by Gerwin2k** [comment]
![no power down 2](https://user-images.githubusercontent.com/39087372/221324741-ba099ef4-c962-4a83-9843-042cc19cf998.png)

I deleted the old rules.ini. Noticing that the game writes a new rules.ini at startup. 
Unfortunately a quick test suggest the situation is unchanged. 
In attached image I just destroyed the only two (non-advanced) power plants in an enemy NOD base. SAM site is still operating. 
(also again two obelisks there, without sufficient power plants to begin with)
